exporter/Cargo.toml

```diff
@@ -8,18 +8,22 @@ authors = ["Sahil Raja <sahil.raja@datacore.com>"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [[bin]]
 name = "metrics-exporter-pool"
-path = "src/bin/pool/main.rs"
+path = "src/main.rs"
```

Comment:
I did the change because the all the code in the root dir, was specific to pool, exporter only. I think in that case we should change the directory structure to this format below. 

```
exporter
├── Cargo.toml
├── README.md
└── src
    ├── lib.rs
    └── pool
        ├── cache.rs
        ├── client
        │   ├── grpc_client.rs
        │   ├── mod.rs
        │   └── pool.rs
        ├── collector
        │   ├── mod.rs
        │   └── pools_collector.rs
        ├── config.rs
        ├── error.rs
        └── main.rs

```